### PR TITLE
fix: try to make kapa work normally by setting CSP header

### PR DESCRIPTION
**Pull request by zll600**
want to fix this problem.

6a7ddfe6-b897-4842-acc1-fd3e4b888c55

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -18,6 +18,7 @@ Options -Indexes
     Header set Access-Control-Allow-Origin: https://apisix.apache.org
     Header set Access-Control-Allow-Methods "*"
     Header set Access-Control-Allow-Headers "*"
+    Header set Content-Security-Policy "frame-src 'self' https://www.google.com https://app.netlify.com"
```
Comment (zll600) [inline .htaccess:21]: Do I need to revert this PR? https://github.com/apache/apisix-website/pull/1773 @SkyeYoung

Comment (SkyeYoung) [reply .htaccess:21]: I think u can just delete the code, and ref the PR here.

Comment (zll600) [reply .htaccess:21]: done

**Review by SkyeYoung — APPROVED** [review]

**Comment by zll600** [comment]
@LiteSun Could you help me review this pr?

**Review by LiteSun — APPROVED** [review]

**Pull request merged**